recipes/sfdc-merlion/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "salesforce-merlion" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1d7e5dcf314cddd30935e8d3d6fbd05641455171461f0c3109b9516cbfa75603
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - cmdstanpy
+    - cython
```

Comment:
I know this is listed in `setup.py`, but it makes no sense to depend on `cython` for a `noarch` package I think--especially since no compiler is installed either.
```suggestion
```

Reply:
Yeah I guess since I took out the pip check this should work